src/components/map/RealtimeLayersService.js

```diff
@@ -10,14 +10,16 @@ goog.require('ga_map_service');
 
   module.provider('gaRealtimeLayersManager', function() {
     this.$get = function($rootScope, $http, $timeout, gaLayerFilters,
-        gaMapUtils, gaUrlUtils) {
+        gaMapUtils, gaUrlUtils, gaLayers) {
 
       var timers = [];
       var realTimeLayersId = [];
       var geojsonFormat = new ol.format.GeoJSON();
 
-      function setLayerSource(layer) {
-        var fullUrl = gaUrlUtils.proxifyUrl(layer.geojsonUrl);
+      function setLayerSource(layer, isNewSource) {
+        var fullUrl = isNewSource ?
+            gaLayers.getLayerProperty(layer.bodId, 'geojsonUrl') :
```

Comment:
Let's leave it as it is not proxified anyway if we're using `admin` URL I am not sure this will be and is always the case. It doesn't hurt.